### DISCUSSION TITLE
Mbtiles maps and AIP info

### DISCUSSION THREAD

**Discussion by manolohpm** [Akaflieg-Freiburg/enroute#339] [General]
Hello.

I have made a custom map from OpenStreetMaps by using QGIS and some free plugins, showing more info needed for VFR flights (drawing villages even small ones). Very easy.

When I use it with Enroute, the info from AIP (CTR, D, R and P areas, airports, etc) is not shown. But Notams are shown. Is this an issue or is the way it should be?

Custom map
![1691005871128](https://github.com/Akaflieg-Freiburg/enroute/assets/136990096/74b01299-f21a-46e9-8187-78e60e2ddfcd)

Custom showing Notams
![Screenshot_2023-08-02-21-53-32-047_de akaflieg_freiburg enroute](https://github.com/Akaflieg-Freiburg/enroute/assets/136990096/5897079e-0204-4523-98d5-96d6e533058b)

Default map
![1691005871132](https://github.com/Akaflieg-Freiburg/enroute/assets/136990096/82c37458-8a0c-4a06-ab82-9f9716c1ed4f)

I understand that default maps are divided in three independent parts:

- base map
- AIP info
- terrain

But are these parts linked only when used with the app maps and not with the custom map?

Thank you. Manolo.

**Comment by kebekus** [comment]
@manolohpm Thank you for your comment. What you experience is the expected behavior. Let me explain:

The map you created with QGIS is a **raster** map. When you load a raster map into Enroute, Enroute assumes it contains all relevant aviation data and does not show any aviation data itself [many countries provide official ICAO maps in raster format]. NOTAMs are different because they change constantly, so NOTAMs are shown in a separate layer on top of every other map.

If you want a base map with more detail, then you need to create an MBTILES file with high-detail **vector** data that follows the standard [OpenMapTiles](https://github.com/openmaptiles/openmaptiles) schema. Vector maps are not easy to create, but you can find read-to-use files here: 

- https://data.maptiler.com/downloads/planet

Best wishes from Germany,

Stefan.

**Comment by kebekus** [comment]
@manolohpm Totally unrelated: I see that the map copyright notice at the bottom of the screen does not fit the screen width. Assuming that you are a native speaker, of Spanish, could you suggest a shorter translation of "Map copyright info" that would fit a narrow screen?

**Comment by manolohpm** [comment]
Sure. Send me all the texts in the app. There are more Spanish texts to be
checked.


Saludos / Best regards
Manolo Hernandez-Peña M.

**Comment by kebekus** [comment]
@manolohpm Sending the texts is difficult because we use a specialized app to manage our translations. If you are interested in helping with the Spanish translations, please send me an e-mail, and I can get you in touch with our translation team. Any help is greatly welcome!
